### ISSUE TITLE
[BUG]:  code indent problem

### ISSUE THREAD

**Issue by 18684092** [KarolineH/projectpage#1]
### Description of the bug

Re: https://github.com/LCAS/LAST-Straw/blob/master/python/data_io.py
In the main function the docstring is indented wrong. (Requires extra space)

```
if __name__ == '__main__':
   
   '''Example usage of the functions defined in this file'''

    # Please specify where the data set is located:
    data_dir = '/home/.../'
```

### Steps To Reproduce

Copy / paste the file into VS Code and intellisense highlights problem

### Additional Information

none

**Comment by gcielniak** [comment]
Thanks, Andy, can you turn this issue into a pull request, please? Fork, make changes that fix the bug and then sync with the main branch.

**Comment by 18684092** [comment]
Yes. There is also another bug as it is hardcoded to only work with the first scan. I'll fix this and do as you say and then will also suggest a pytorch dataloader version which I am writing now.
